lib/form/html.js

```diff
@@ -189,7 +189,7 @@ const renderer = (type, errors) => {
 
           // Adds delete button if instances are greater than minimum
           if (input.inputs.length > input.repeatable.min) {
-            render += `<button type="button" class="delete--button" id="${input.id}--delete--${index}">Delete</button>`;
+            render += `<button type="button" class="form--delete" id="${input.id}--delete--${index}">Delete</button>`;
```

Comment:
Is the JS looking for `form--delete` or `delete--button`? In the [HTML generation in Content Types](https://github.com/punchcard-cms/content-types/blob/master/lib/form/html.js#L192) it's adding `delete--button` but it _looks_ like the [JS is looking for `form--delete`](https://github.com/punchcard-cms/content-types/blob/master/lib/form/js/repeatable.js#L196). 

Can you confirm which is which and update whichever is wrong accordingly? It also looks like we may need to refactor that JS a bit.